cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -853,6 +854,124 @@
     </description>
   </property>
 
+  <property>
+    <name>kafka.bind.address</name>
```

Comment:
These properties should be kept in alphabetical order in the section, so that when the docs are created, they can be looked up alphabetically. You can see the output here: http://docs.cask.co/cdap/develop/en/admin-manual/appendices/cdap-site.html#appendix-cdap-site-xml which is generated directly from this file.